CMakeLists.txt

```diff
@@ -542,6 +542,7 @@ FIND_PACKAGE(Qt${QT_VERSION_MAJOR} COMPONENTS Charts REQUIRED)
 IF(USE_PLUGIN_TELESCOPECONTROL OR ENABLE_GPS)
      FIND_PACKAGE(Qt${QT_VERSION_MAJOR} COMPONENTS SerialPort REQUIRED)
 ENDIF()
+FIND_PACKAGE(ShowMySky REQUIRED)
```

Comment:
As much as I will like to use it on my PC, I think this should be an optional build component. Just testing regular build on an old Odroid C1 (Ubuntu Mate 20.04LTS (upgrade to offered 22.04 failed), Qt5 only, GL ES2 on Arm SBC with Mali GPU). If anything goes wrong, we should be able to build without ShowMySky. Please add a CMAKE config variable and depending code exclusions. (Many examples in this file.)